src/peergos/shared/crypto/password/PasswordProtected.java

```diff
@@ -31,7 +30,7 @@ public static Cborable encryptWithPassword(byte[] cleartext,
                                                Salsa20Poly1305 provider,
                                                SafeRandom random) {
         List<Cborable> elements = new ArrayList<>();
-        byte[] saltBytes = random.randomBytes(TweetNaCl.SECRETBOX_KEY_BYTES);
+        byte[] saltBytes = random.randomBytes(32);
```

Comment:
use of constant

Reply:
This has nothing to do with secretbox, and isn't used anywhere else.